em_workflows/utils/utils.py

```diff
@@ -447,13 +447,15 @@ def list_dirs(input_dir_fp: Path) -> List[Path]:
 
 
 @task(retries=1, retry_delay_seconds=10)
-def notify_api_running(no_api: bool, token: str, callback_url: str):
+def notify_api_running(no_api: bool=None, token: str=None, callback_url: str=None):
```

Comment:
I believe, the original function definition is still better. Because everything has to be explicitly passed rather than relying on defaults.

Reply:
We shall get back to this later!